sdk/keyvault/arm-keyvault/test/keyvault.spec.ts

```diff
@@ -64,7 +64,7 @@ describe("Keyvault test", () => {
     for await (const item of client.operations.list()) {
       resArray.push(item);
     }
-    assert.ok(resArray.length);
+    assert.isAbove(resArray.length, 0);
```

Comment:
are these ARM tests generated or handwritten?

Reply:
This looks manually added. The latest generated test only contains simpler one, which needs to be fixed too:
```ts
  it("operation list test", async () => {
    const result = await client.operations.list();
    assert.ok(result);
  });
```